### ISSUE TITLE
Shutting down windows

### ISSUE THREAD

**Issue by jespertheend** [AlexGustafsson/homebridge-wol#93]
I was having a lot of trouble getting windows 10 to shut down.
After tinkering with firewalls etc. I finally managed to find a super easy solution.
Install [this application](https://github.com/karpach/remote-shutdown-pc) on your pc.
As shutdown command you can then use `curl http://192.168.1.32:5001/secret/shutdown`

Also make sure to enable File and printer sharing in your firewall otherwise the ping doesn't work.

I hope that info saves some time.

**Comment by AlexGustafsson** [comment]
Thanks a lot for the information. I’ll add it to the README for the sake of visibility. I’m sure it’ll help some Windows users.

**Comment by DM164** [comment]
I tried using this method and it works when I open the URL on my PC (the one I'm trying to shutdown) but when I try from any other device it won't connect. I also allowed connections in on my Firewall for File and printer sharing but it doesn't seem to work. 
![image](https://user-images.githubusercontent.com/41427432/103406913-b1620e80-4b5c-11eb-9a4b-83bb76034b90.png)
Do I have to change anything in here?

**Comment by jespertheend** [comment]
In case anyone stumbles upon this, I was running into issues where sometimes the computer wouldn't properly shut down.
This was caused by some applications preventing shutdown under certain circumstances. I have now configured ssh on my windows machine and my configuration currently looks like:

```json
"shutdownCommand": "ssh Jesper@192.168.1.2 shutdown /s /t 1"
```

This forces windows to shutdown regardless of any open applications.

You'll have to make sure your homebridge user has a ssh key generated though. In my case I did that using `ssh-keygen -t RSA` (while logged in as the homebridge user using `su homebridge`). I had to create the `/home/homebridge/` directory for this to work, because it didn't exist yet.
